platform/security/servlet/security-servlet-logout/src/test/java/org/codice/ddf/security/servlet/logout/TestLogoutServlet.java

```diff
@@ -33,40 +33,81 @@
 import javax.servlet.http.HttpSession;
 import org.apache.shiro.subject.Subject;
 import org.apache.shiro.util.ThreadContext;
+import org.junit.Before;
 import org.junit.Test;
 
 public class TestLogoutServlet {
-  @Test
-  public void testLocalLogout() {
+  LocalLogoutServlet localLogoutServlet;
```

Comment:
Any reason these aren't private?

Reply:
No reason. Though since most of our existing tests look like they make them private I will change them. 
We should add something to our unit testing guidelines to specify since I don't think we have anything on this